raft/raft.hh

```diff
@@ -324,7 +324,7 @@ struct no_other_voting_member : public error {
 };
```

Comment:
Please add
```
Fixes: scylladb/scylladb#18902
```
to the commit message.

For single-commit PRs like this, the PR cover letter is lost during merge. (Because only add the single commit to the final git log.)

Reply:
Done